workflows/proteomics/clinicalmp/clinicalmp-discovery/iwc-clinicalmp-discovery-workflow-tests.yml

```diff
@@ -0,0 +1,22 @@
+- doc: Test outline for iwc-clinicalmp-discovery-workflow
+  job:
+    Human UniProt Microbial Proteins from MetaNovo and cRAP:
+      class: File
+      location: https://zenodo.org/records/10720030/files/Human_UniProt_Microbial_Proteins_(from_MetaNovo)_and_cRAP.fasta
+      filetype: fasta
+    Experimental Design Discovery MaxQuant:
+      class: File
+      path: test-data/Experimental Design Discovery MaxQuant.tabular
+      filetype: tabular
+    Tandem Mass Spectrometry MSMS files:
+      class: Collection
+      collection_type: list
+      elements:
+      - class: File
+        identifier: PTRC_Skubitz_Plex2_F10_9Aug19_Rage_Rep-19-06-08.raw
+        location: https://zenodo.org/records/14182981/files/PTRC_Skubitz_Plex2_F10_9Aug19_Rage_Rep-19-06-08.raw
+  outputs:
+    SGPS MQ Peptides:
+      asserts:
+        - has_n_columns:
```

Comment:
Could you add some more specific checks here ?

Reply:
The output can be different based on databases being added, so I thought there should be a column at least. Should I specify rows?